Crescent-Labs/iqra-android
# Added a translation bar to the results screen

### Design 

![iqra](https://cloud.githubusercontent.com/assets/8183131/20641530/1f84a124-b3c8-11e6-9361-4a5ada2b3289.png)

### What are you trying to accomplish?

I am adding the translation bar into Iqra.

### How are you accomplishing this?

My first goal was to organize fundamental networking logic into a `RequestDelegate` that we can use in `SearchResultsActivity`. Upon refactoring code in `MainActivity` and `SearchResultsActivity`, I added the translation spinner into the layout. 

I hooked the translation spinner to the necessary networking calls required to fetch new translations when the translation changes.

Resolves #5 

### Testing Steps

1. Recite a verse
2. Go to the results page and observe the new translation spinner
3. Change the translation and observe if the results translations change and the setting preferences change
4. Repeat 1-3 until satisifed

- [ ] Unit Tests (not needed until we refactor the app more)
